config/desktop.json

```diff
@@ -82,6 +82,7 @@
 		"settings/theme-setup": false,
 		"signup/domain-first-flow": true,
 		"signup/social": false,
+		"signup/wpcc": true,
```

Comment:
Is it really a flow that can happen in the desktop app @scruffian?

Reply:
There's no way to get to it in the desktop app AFAIK, but it would work in the desktop app if it was exposed, so I don't see a reason to have a different behavior there.

Reply:
Well my reasoning is that by enabling it we let other developers wonder why it's enabled if it's not used, but I see your point.

Reply:
Makes sense, we could simply remove it here